src/about-page/styles.css

```diff
@@ -0,0 +1,120 @@
+ * {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: none;
+  background: none;
+} 
+.main-container{
+  width: 100%;
+  height: 1000px;
+  display: flex;
+  flex-direction: column;
+  border: solid 1px pink;
+  flex-grow: 1;
+
+}
+.header{
+  width: 100%;
+  height: 200px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  background-color: salmon;
+  font-family: 'Lato', sans-serif;
+  color: rgb(197, 95, 112);
+}
+.header h1{
+  font-size: 35px;
+}
+.header i{
+  font-family: 'Amatic SC', cursive;
+  color: slategrey;
+  font-size: 50px;
+}
+.header h4{
+  font-size: 15px;
+  font-family: 'Lato', sans-serif;
+  color: ghostwhite
+}
+.story{
+  width: 100%;
+  height: 5fr;
+  padding-left: 150px;
+  padding-right: 150px;
```

Comment:
i think this is a lil too much padding but you could have done it for a reason! try just 20px. up to you!